libexec/scoop-search.ps1

```diff
@@ -9,35 +9,43 @@ param($query)
 . "$PSScriptRoot\..\lib\manifest.ps1" # 'manifest'
 . "$PSScriptRoot\..\lib\versions.ps1" # 'Get-LatestVersion'
 
+$list = @()
+
+try {
+    $query = New-Object regex $query, 'IgnoreCase'
```

Comment:
```suggestion
    $query = New-Object Regex $query, 'IgnoreCase'
```

Reply:
Tab-completion shows `regex`, so I did not capitalize it

Reply:
The class name is [System.Text.RegularExpressions.Regex](https://docs.microsoft.com/en-us/dotnet/api/system.text.regularexpressions.regex?view=netframework-4.5) and since it's under System.Text, a single 'Regex' is okay.

This is just, hmm, convention, right?

Reply:
I mean it doesn't really matter, but as a rule of thumb I go with what the auto-completion tells me 😅

Reply:
Yes, the same, since Windows really doesn't care about cases, so does PowerShell.

I mentioned it and `$_.Exception.InnerException.Message` above just for 'formal', you know. 😄